filters/include/pcl/filters/functor_filter.h

```diff
@@ -9,24 +9,28 @@
 
 #pragma once
 
+#define PCL_NO_PRECOMPILE
+#include <pcl/filters/filter_indices.h>
+#undef PCL_NO_PRECOMPILE
```

Comment:
You're unconditionally defining PCL_NO_PRECOMPILE and then undefining it. If a user defines it in his/her code and then includes this filter, we're gonna mess up his/her definitions. 

tl;dr: remember the user can define PCL_NO_PRECOMPILE before you

Reply:
Fixed with 98e44f7